tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -3944,6 +3944,69 @@ function ( $payment_information ) use ( $payment_method_id, $express_payment_met
+			->willReturn( $mock_intent );
+
+		// Before processing the order has no transaction ID.
+		$this->assertSame( '', $order->get_transaction_id() );
+
+		$response = $this->mock_gateway->process_payment( $order_id );
+
+		$this->assertEquals( 'success', $response['result'] );
+
+		$reloaded = wc_get_order( $order_id );
+		$this->assertSame( 'ch_mock1234567890', $reloaded->get_transaction_id() );
+	}
```

PHPMD (2.15.0)

[error] test_process_payment_with_confirmation_token_backfills_missing_transaction_id accesses the super-global variable $_POST. (undefined)

(Superglobals)